pkg/target/kusttarget.go

```diff
@@ -347,3 +348,23 @@ func (kt *KustTarget) getAllVars() ([]types.Var, error) {
 	}
 	return result, nil
 }
+
+func loadKustFile(ldr ifc.Loader) ([]byte, error) {
+	content, err := ldr.Load(constants.KustomizationFileName)
+	if err == nil {
+		return content, nil
+	}
+	if !strings.Contains(err.Error(), "no such file or directory") {
+		return nil, err
+	}
+
+	content, nerr := ldr.Load(constants.SecondaryKustomizationFileName)
```

Comment:
content, err = 

no need for new vars.

The error string check could be placed in a function so its not repeated.

Reply:
I put the repeated code into a for loop.